flow-jcloud.yml

```diff
@@ -30,10 +30,12 @@ executors:
     uses_with:
       name: ViT-L-14-336::openai
     replicas: 1
-    host: 'demo-cas.jina.ai'
+    host: 'api.clip.jina.ai'
     port: 2096
-    tls: true
-    external: true
+    tls: false
+    external: false
```

Comment:
why tls & external = false here

Reply:
In this PR, the executor is hosted by the user defaultly. To use clip-as-service, the user need update the yaml by themselves.